package_manifest.yaml

```diff
@@ -721,18 +721,10 @@ foreman_plugin_packages:
   children:
     ruby_plugin_packages: {}
   hosts:
-    ansible-collection-theforeman-operations: {}
-    ansiblerole-foreman_scap_client: {}
-    ansiblerole-insights-client: {}
     ansible-runner: {}
```

Comment:
Isn't runner also only for the plugin?

Reply:
yes, but I didn't want to build it just yet, so left it out for now